crates/lang/ir/src/ir/item_impl/callable.rs

```diff
@@ -221,7 +221,7 @@ pub trait Callable {
 /// }
 /// ```
 ///
-/// ... then the selector of `my_message` is simply `0xDEADBEEF` since it overrides
+/// …then the selector of `my_message` is simply `0xDEADBEEF` since it overrides
```

Comment:
I'd still get rid of the ellipsis unicode character. Tbh I don't even know what key combination to press to plop one out. I'm sure I could figure it out (or copy the existing ones in the code), but keeping three periods makes our life easier when writing docs. Secondly, if you're worried about rendering, any clever enough font (e.g [FiraCode](https://github.com/tonsky/FiraCode/)) will be able to produce nice ligatures out three periods (or `->` as you mentioned during the last round of review).

Reply:
>makes our life easier when writing docs

This shouldn't be the goal, we should aim for the best user experience, not for our best experience when writing code.

>any clever enough font (e.g FiraCode) will be able to produce nice ligatures out three periods

You linked a mono space font, but the changes I am making are in for the non-monospace rustdoc-generated HTML and there is no font which will modify the generated HTML into some other symbol. Ligatures are about making characters which appear next to each other appear nicely together (e.g. fi), they're not about merging characters into a different symbol.

Reply:
If the developers don't have a good experience while writing docs, docs aren't going to get written, and the users lose either way 😉